packages/select/select.scss

```diff
@@ -68,7 +68,9 @@ $select-error-color--inverted: $feil--darkbg;
             font-weight: normal;
             font-family: sans-serif;
             color: $select-text-color;
-            color: var(--select-text-color);
```

Comment:
Hva er grunnen til at denne er tatt vekk? Tipper det vi ødelegge darkmode om man bruker data-theme isteden for inverted-propen.

Reply:
Mulig jeg tar feil altså, men jeg tok den vekk fordi denne blokka, viser native select option elementer, og der bør man strengt tatt ikke invertere til hvit tekst i darkmode. Dette slår litt stygt ut som hvit tekst i Firefox (men ikke i Chrome) og det slår til som hvit tekst med hvit bakgrunn i Chrome på Linux. Ergo umulig å se. Så derfor tenkte jeg det var best at den uansett settes til select-text-color variabel som er svart. Okke som.

Reply:
Aha, forstår :) Da gir det mening.